cdk/dcm_floodlight/transform.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-param-reassign */
+const get = require("get-value");
+const {
+  getValueFromMessage,
+  removeUndefinedAndNullValues,
+  isDefinedAndNotNull
+} = require("rudder-transformer-cdk/build/utils");
+const { CustomError, getIntegrationsObj, isEmpty } = require("../../v0/util");
+
+// append properties to endpoint
+// eg: ${endpoint}key1=value1;key2=value2;....
+function appendProperties(endpoint, payload) {
+  Object.keys(payload).forEach(key => {
+    endpoint += `${key}=${payload[key]};`;
+  });
+
+  return endpoint;
+}
+
+// transform webapp dynamicForm custom floodlight variable
+// into {u1: value, u2: value, ...}
+function transformCustomVariable(customFloodlightVariable) {
+  const customVariable = {};
+  customFloodlightVariable.forEach(item => {
+    if (!isEmpty(item.from) && !isEmpty(item.to)) {
```

Comment:
Check validity of item before accessing its propery

Reply:
if the `item` is not present then forEach simply won't traverse

Reply:
Tested like some its `traversing`

Reply:
dynamicForm values are always defined